script/States/App/LTheory.lua

```diff
@@ -19,52 +22,60 @@ function LTheory:generate ()
 
   if self.system then self.system:delete() end
   self.system = System(self.seed)
+
```

Comment:
Do we need this file still? And if we do, we should rename it to its purpose.

Reply:
I had a very quick look at Quickstart. I couldn't tell if it was using the values from user.ini or not if it is, that makes changing values for quick testing harder (and affecting LTheoryRedux) than LTheory. I'd be OK with renaming LTheory if you're convinced there's some value in doing that, but if Quickstart is using user.ini values, then I'd like to keep LTheory as I do find it very useful.

Reply:
QuickStart uses LTheoryRedux to create the application but without going into the main menu :) so everything that is changed in LTheoryRedux.lua will also be changed in QuickStart automatically / it will use the user config